app-android/src/main/java/io/github/droidkaigi/confsched2022/KaigiApp.kt

```diff
@@ -174,6 +175,10 @@ fun KaigiApp(
                 )
             }
         }
+
+        sessionIdFromNotification?.let {
+            kaigiAppScaffoldState.onTimeTableClick(TimetableItemId(it))
+        }
```

Comment:
I think this `onTimeTableClick` can be called many times. How about using Launched Effect?

```suggestion
LaunchedEffect(sessionIdFromNotification) {
  sessionIdFromNotification?.let {
    kaigiAppScaffoldState.onTimeTableClick(TimetableItemId(it))
  }
}
```